<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QMainWindow" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt ONI Viewer - alpha test</string>
  </property>
  <widget class="QWidget" name="center">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="grid">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="time_slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QFrame" name="butt_frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>55</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="buttons">
           <item row="0" column="1" rowspan="2">
            <widget class="QPushButton" name="first_frame">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>First Frame</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6" rowspan="2">
            <widget class="QPushButton" name="last_frame">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Last Frame</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4" rowspan="2">
            <widget class="QPushButton" name="next_frame">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Next Frame</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="play_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Play</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="QPushButton" name="prev_frame">
             <property name="minimumSize">
              <size>
               <width>82</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>82</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Previous Frame</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="pause_button">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <spacer name="right_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="7">
            <widget class="QLabel" name="right_label">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>00:00:00 (F0)</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="left_spacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="left_label">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="left_gview"/>
      </item>
      <item row="0" column="1">
       <widget class="QGraphicsView" name="right_gview"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
